README.md

```diff
@@ -1 +1,63 @@
 # jQuery Number Input Formatter
+
+A jQuery library to dynamically display a numeric input's formatted value in a
+label.
+
+This library will take the value of an HTML input, format it, and place the
+formatted representation into a label. The purpose is to give the end user an
+opportunity to more easily identify their numeric inputs for large numbers.
+
+For example, an input value of `12345.678` will be displayed on the page as
+`12,345.678`.
+
+## Usage
+
+1. Include jQuery
+
+```html
+<script type="text/javascript" src="http://code.jquery.com/jquery.js"></script>
+```
+
+2. Include jQuery Number Input Formatter
+
+```html
+<script src="dist/jquery.environmentbar.min.js"></script>
+```
+
+3. Find an input that is expecting a numeric input
+
+```html
+<input class="number-input" value="12345.678">
+```
+
+4. Add a label to it
+
+```html
+
+<!-- this library will detect a span label by default using $.next -->
+<input class="number-input" value="12345.678">
+<span></span>
+
+<!-- if you specify a selector in the options, any element may be used and found using $.next -->
+<input class="number-input" value="12345.678">
+<div class="custom-label"></div>
+```
+
+5. Initialize the formatter
+ 
+```html
+<script>
+// default when using a span as the label
+$('.number-input').numberInputFormatter();
+
+// if the label is something other than a span next
+$('.number-input').numberInputFormatter({inputSelector: '.custom-label'});
+</script>
+```
+
+## Options
+
+You may provide options into the library to alter its behaviour.
+
+- `inputSelector` - a CSS selector to identify the label on the page on which
```

Comment:
Might actually steer clear of using the word "label" for this. I assumed at first it had something to do with a `<label>` since we're talking about `<input>`s. Maybe confirmation text or something?